libcloud/compute/drivers/vultr.py

```diff
@@ -119,6 +133,15 @@ def create_node(self, name, size, image, location):
 
         return created_node
 
+    def ex_list_ssh_keys(self):
```

Comment:
Any particular reason why this is implemented as an extension method and not as a standard `list_key_pairs` method? (https://github.com/apache/libcloud/blob/trunk/libcloud/compute/base.py#L1160)

Looking at the code, it looks like you should be able to implement it using `list_key_pairs` method which is part of the standard API (use name for the id attribute, you can leave fingerprint as None or calculate it from the public key).